controllers/C_InsuranceCompany.class.php

```diff
@@ -69,77 +68,11 @@ function edit_action_process() {
 		$this->icompanies[0]->persist();
 		$this->icompanies[0]->populate();
 
-		// Post insurance companies as customers to the accounting system
-		// unless globals.php requests otherwise.
-		//
-		if (! $GLOBALS['insurance_companies_are_not_customers'])
```

Comment:
This makes me think; can the entire integration_mapping table go since this seems to be an old sql-ledger thing?
@sunsetsystems ?